### PR TITLE
Sync the OSF sponsorship page with the current sponsors

### PR DESCRIPTION
**Pull request by iamamoose**
Sync to the current list of OSF sponsors.  Add a bronze level and the current sponsors at that level.
Add a link to the 'in kind' thanks page. Add a new section to call out our past sponsors.

### CODE REVIEW

**Review by t-j-h — APPROVED** [review]

File: support/acks.html [hunk 1]
```diff
@@ -7,19 +7,19 @@
...
-	  for their significant support of the OpenSSL project. Sponsors are
-	  listed alphabetically within categories.  Please note that we ask
-	  permission to identify sponsors and that some sponsors we consider
-	  eligible for inclusion here have requested to remain anonymous.</p>
+          <h3 id="current">Sponsorship Donations</h3>
 
-          <h3 id="current">Current Sponsors:</h3>
+
+	  <p>We would like to identify and thank the following sponsors
+	  for their donations which give significant support of the OpenSSL project.
```
Comment (mattcaswell) [inline support/acks.html]: Should that be *to* the OpenSSL project?

**Review by mattcaswell — CHANGES_REQUESTED** [review]

**Review by mattcaswell — APPROVED** [review]

**Comment by mspncp** [comment]
Note that this pull request was merged via (non fast-forward) merge commit. Was this done accidentally, or are merge commits allowed in this repository? If not, do we care to fix it?

```
*   0f13e11 (HEAD -> master, origin/master, github/master, github/HEAD) Merge pull request #140 from iamamoose/sponsorship
|\  
| * 5fbd49f (github/gh-140) Better grammar for where the support goes
| * cea0496 Update the sponsorship page to be current with the list of OSF sponsors.  Add a bronze level and the current sponsors at that level. Add a link to the 'in kind' thanks page.
|/  
* 8930b3a Add a link to the new blog post
```

**Comment by mattcaswell** [comment]
Gitolite controls where merge commits are allowed. They will be prevented in the source repo as well as a number of the other ones such as tools and the book. But the web one does not have this rule. I'm not sure whether that is by accident or design.

**Comment by mspncp** [comment]
Never mind, it's your (the OMC's) decision. I just wanted to give you a heads-up.

**Pull request merged**